runtime/cudaq/platform/default/rest/helpers/oqc/OQCServerHelp.cpp

```diff
@@ -123,7 +126,25 @@ void OQCServerHelper::initialize(BackendConfig config) {
   config["password"] = make_env_functor("OQC_PASSWORD")();
   // Construct the API job path
   config["job_path"] = "/tasks"; // config["url"] + "/tasks";
-                                 //
+
+  // Parse the output_names.* (for each job) and place it in outputNames[]
+  for (auto &[key, val] : config) {
+    if (key.starts_with("output_names.")) {
+      // Parse `val` into jobOutputNames.
+      // Note: See `FunctionAnalysisData::resultQubitVals` of
+      // LowerToBaseProfileQIR.cpp for an example of how this was populated.
+      OutputNamesType jobOutputNames;
+      nlohmann::json outputNamesJSON = nlohmann::json::parse(val);
+      for (const auto &el : outputNamesJSON[0]) {
+        std::size_t result = el[0].get<std::size_t>();
+        std::size_t qubit = el[1][0].get<std::size_t>();
```

Comment:
nit: Don't need to repeat the type on LHS and RHS.

Reply:
Updated in c4fc0b8a